libp2p/protocols/identify.nim

```diff
@@ -69,7 +70,7 @@ proc decodeMsg*(buf: seq[byte]): IdentifyInfo =
   result.pubKey = none(PublicKey)
   var pubKey: PublicKey
   if pb.getValue(1, pubKey) > 0:
-    trace "read public key from message", pubKey = $pubKey
+    trace "read public key from message", pubKey = ($pubKey).shortLog
```

Comment:
probably the type of the pubkey should have its own shortlog format rather than using the string conversion first